files/bash_history_timestamp.sh

```diff
@@ -0,0 +1,2 @@
+#append date & time stamps
+export HISTTIMEFORMAT="%F %T "
```

Comment:
Lets also make the history file way way longer here please.

Reply:
I thought of making the history file longer, but then it may conflict with what other people have customly set.  Do we care about that?

Reply:
Should i set $HISTSIZE = 5000?

Reply:
Lets say 10k